wpiformat/wpiformat/cidentlist.py

```diff
@@ -117,7 +125,9 @@ def run_pipeline(self, config_file, name, lines):
             elif token == "\\'":
                 continue
             elif token == "'":
-                if not in_string and match.start() != last_digit_end:
+                if not in_string and not (
+                    not in_char and self.is_quote_in_number(lines, match.start())
+                ):
```

Comment:
I had an easier time reading it like this:
```py
                if in_string:
                    continue
                if in_char or not self.is_quote_in_number(lines, match.start()):
                    in_char = not in_char
```
Makes it clearer the "in string" check is unrelated.

Reply:
What about this?
```python
                if in_string:
                    continue
                if not in_char and self.is_quote_in_number(lines, match.start()):
                    continue
                in_char = not in_char
```
That way, the "in string" check and the "is in number" check are structured the same way.

Reply:
Sounds good to me.